Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -1001,6 +1010,149 @@ found, it is used.
 
+.. _Neos_Neos__MenuItems:
+
+Neos.Neos:MenuItems
+-------------------
+
+Create a list of menu-items items for nodes.
+
+:entryLevel: (integer) Start the menu at the given depth
+:maximumLevels: (integer) Restrict the maximum depth of items in the menu (relative to ``entryLevel``)
+:startingPoint: (Node) The parent node of the first menu level (defaults to ``node`` context variable)
+:lastLevel: (integer) Restrict the menu depth by node depth (relative to site node)
+:filter: (string) Filter items by node type (e.g. ``'!My.Site:News,Neos.Neos:Document'``), defaults to ``'Neos.Neos:Document'``
+:renderHiddenInIndex: (boolean) Whether nodes with ``hiddenInIndex`` should be rendered, defaults to ``false``
+:itemCollection: (array) Explicitly set the Node items for the menu (alternative to ``startingPoints`` and levels)
+
+MenuItems item properties:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:node: (Node) A node instance (with resolved shortcuts) that should be used to link to the item
+:originalNode: (Node) Original node for the item
+:state: (string) Menu state of the item: ``'normal'``, ``'current'`` (the current node) or ``'active'`` (ancestor of current node)
+:label: (string) Full label of the node
+:menuLevel: (integer) Men^u level the item is rendered on
+
+Examples:
+^^^^^^^^^
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		entryLevel = 1
+		maximumLevels = 3
+	}
+
+MenuItems including site node:
+""""""""""""""""""""""""""""""
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		itemCollection = ${q(site).add(q(site).children('[instanceof Neos.Neos:Document]')).get()}
+	}
+
+Menu with custom starting point:
+""""""""""""""""""""""""""""""""
+
+::
+
+	menuItems = Neos.Neos:MenuItems {
+		entryLevel = 2
+		maximumLevels = 1
+		startingPoint = ${q(site).children('[uriPathSegment="metamenu"]').get(0)}
+	}
+
+.. _Neos_Neos__BreadcrumbMenuItems:
+
+Neos.Neos:BreadcrumbMenuItems
+-----------------------------
+
+Create a list of of menu-items for a breadcrumb (ancestor documents), based on :ref:`Neos_Neos__MenuItems`.
+
+Example::
+
+	breadcrumbItems = Neos.Neos:BreadcrumbMenuItems
+
+.. _Neos_Neos__DimensionsMenuItems:
+
+Neos.Neos:DimensionsMenuItems
+-----------------------------
+
+Create a list of menu-items for other node variants (e.g. variants of the current node in other dimensions) by using this Fusion object.
+
+If the ``dimension`` setting is given, the menu will only include items for this dimension, with all other configured
+dimension being set to the value(s) of the current node. Without any ``dimension`` being configured, all possible
+variants will be included.
+
+If no node variant exists for the preset combination, a ``NULL`` node will be included in the item with a state ``absent``.
+
+:dimension: (optional, string): name of the dimension which this menu should be based on. Example: "language".
+:presets: (optional, array): If set, the presets rendered will be taken from this list of preset identifiers
```

Comment:
nitpick2: Could we maybe link this to the corresponding [documentation](https://neos.readthedocs.io/en/latest/CreatingASite/ContentDimensions.html#preset-constraints) or adjust the name so that it is a bit clearer what "presets" refer to
```suggestion
:presets: (optional, array): If set, the Content Dimension Preset Constraints will be taken from this list of preset identifiers
```

Reply:
Same here, let us update this separately as this should also be improved for the documentation of ``Neos.Fusion:Menu``.